cmd/gossamer/commands/root.go

```diff
@@ -162,9 +168,13 @@ func addRootFlags(cmd *cobra.Command) error {
 	}
 
 	// Log Config
-	if err := addLogFlags(cmd); err != nil {
-		return fmt.Errorf("failed to add log flags: %s", err)
-	}
+	cmd.Flags().StringVarP(&logLevel, "log", "l", "",
+		`Set a logging filter.
+	Syntax is a list of 'module=logLevel' (comma separated)
+	e.g. --log sync=debug,core=trace
+	Log levels (least to most verbose) are error, warn, info, debug, and trace.
+	By default, all modules log 'info'.
+	The global log level can be set with --log global=debug`)
```

Comment:
Let's add the list of available modules in the usage https://github.com/ChainSafe/gossamer/issues/3210#issuecomment-1511518468